WordPress/gutenberg
# Update Global Styles code to continue adding settings & styles

Related thread https://github.com/WordPress/gutenberg/pull/38039#discussion_r808702086

This PR renames the `wp_get_global_*` functions to `gutenberg_get_global_*` and updates all the instances in which we use them in the plugin. The rationale for this change is that we're not done adding settings & styles to `theme.json` and we need the Gutenberg plugin to keep iterating while supporting WordPress 5.8 and WordPress 5.9.

Examples of PRs in which we may need these are https://github.com/WordPress/gutenberg/pull/38039 https://github.com/WordPress/gutenberg/pull/32499 https://github.com/WordPress/gutenberg/pull/37770 https://github.com/WordPress/gutenberg/pull/32502 https://github.com/WordPress/gutenberg/pull/38319 Because there're so many of them and we don't know which PR will land first, I've gone ahead and prepared this PR to fast-track these changes so the PRs above can focus on feature work.

## How to test

Automated tests should pass. Load a block-base theme and verify that it works as expected.
